js/everything.js

```diff
@@ -337,26 +317,22 @@ var ZenPlayer = {
             toggleElement(event, ".plyr__video-wrapper", "Player");
         });
     },
-    setupAutoplayToggle: function()
-    {
+    setupAutoplayToggle: function() {
         // toggle auto next song playing
-        $("#toggleAutoplay").click(function(event)
-        {
+        $("#toggleAutoplay").click(function(event) {
             var toggleTextElement = $("#" + event.currentTarget.id);
-            if (autoplayState === null)
-            {
-                autoplayState = "true";
+            if (!autoplayState) {
+                autoplayState = true;
                 toggleTextElement.text("Stop autoplay");
             }
             else {
-                if (autoplayState === "true")
-                {
+                if (autoplayState === true) {
                     toggleTextElement.text("Start autoplay");
-                    autoplayState = "false";
+                    autoplayState = false;
                 }
                 else {
```

Comment:
When does this else branch ever run? We're already checking for `!autoplayState` in the first if branch. Looks like we can simplify this to a single if/else